src/Context/WebApiContext.php

```diff
@@ -269,9 +293,9 @@ public function printResponse()
         echo sprintf(
             "%s %s => %d:\n%s",
             $request->getMethod(),
-            $request->getUrl(),
+            $request instanceof RequestInterface ? (string) $request->getUri() : $request->getUrl(),
             $response->getStatusCode(),
-            $response->getBody()
+            $request instanceof RequestInterface ? (string) $response->getBody() : $response->getBody()
```

Comment:
btw, checking the response would be better than checking the request for something involving the response